esp-wifi/MIGRATING-0.10.md

```diff
@@ -42,3 +42,42 @@ The cost of this is that we need to rename the various `async` methods on `WifiC
 - controller.start().await.unwrap();
 + controller.start_async().await.unwrap();
 ```
+
+## The blocking networking stack was removed
+
+The blocking networking stack is not included anymore. You can use e.g. `smoltcp-nal` or even use `smoltcp` directly.
+
+For an easy migration path there is https://github.com/bjoernQ/blocking-network-stack.git which is basically the previously included networking stack as it's
+own crate.
+
+The `create_network_interface` function doesn't take `&mut SocketSet[..]` anymore.
+
+```diff
++use blocking_network_stack::Stack;
+use esp_wifi::{
+-    wifi_interface::WifiStack,
+};
+
+-    let init = init(
+-        timg0.timer0,
+-        Rng::new(peripherals.RNG),
+-        peripherals.RADIO_CLK,
+-    )
+-    .unwrap();
++    let mut rng = Rng::new(peripherals.RNG);
++    let init = init(timg0.timer0, rng.clone(), peripherals.RADIO_CLK).unwrap();
```

Comment:
The diff is a little hard to read. This should make the "What's changed" clearer
```suggestion
+    let mut rng = Rng::new(peripherals.RNG);
-    let init = init(timg0.timer0, rng, peripherals.RADIO_CLK).unwrap();
+    let init = init(timg0.timer0, rng.clone(), peripherals.RADIO_CLK).unwrap();
```